starter-code/css/main.css

```diff
@@ -1,8 +1,181 @@
-/*
-Colors:
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
 
-Text: 1A1A1A
-Green: #00B172
-White: #FFF
+body {
+  font-family: "Montserrat", sans-serif;
+}
 
-*/
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 8px;
+  font-size: 1.4em;
+  position: fixed;
+  background-color: white;
+  width: 100vw;
+}
+
+.logo img {
+  width: 200px;
+}
+
+.navbar a {
+  padding: 20px;
+  text-decoration: none;
+  color: black;
+}
+
+header {
+  background-image: url(/lab-css-spotify-clone/starter-code/images/landing.jpg);
```

Comment:
Leaving a path like this should be avoided as it can easily cause errors like the one that I currently have when opening your project where the whole image is gone. I also believe (not 100% sure though) that this can in some cases cause security breaches as people can see where in the folders the different data has been stored.
In the future: Write all local urls like this: background-image: url("./images/landing.jpg"); and remember: ../ means go up one directory and ./ means go down one directory